lib/python/ray/worker.py

```diff
@@ -422,13 +422,18 @@ def put_object(self, objectid, value):
     try:
       buff = self.plasma_client.create(objectid.id(), size, bytearray(schema))
     except RuntimeError as e:
-      if e.args != ("an object with this ID could not be created",):
-        raise
-      # The object already exists in the object store, so there is no need to
-      # add it again. TODO(rkn): We need to compare the hashes and make sure
-      # that the objects are in fact the same.
-      print("This object already exists in the object store.")
-      return
+      if e.args == ("An object with this ID already exists in the plasma store.",):
```

Comment:
I think in Python it's generally nicer to subclass `Exception` with custom exception classes and use different `except` statements, instead of comparing the args.